src/rfac/rfrdargs.c

```
@@ -0,0 +1,366 @@
+    real  s_step               - step for shift
+    int all_groups             - flag: print R-factors of all group ID's
+    real vi                    - im. part of optical potential.
+    real ignore_max            - ignore features larger than ignore_max.
+  */
+
+  rfac_args *args = rfac_args_init();
+
+  /*********************************************************************
+   * decode arguments
+   *********************************************************************/
+  if( argc == 1)
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)